contracts/Governance.sol

```diff
@@ -796,11 +854,20 @@ contract Governance is Context, Ownable, ERC165, EIP712 {
         Proposal storage proposal = _proposals[proposalId];
         bool reached = true;
         for (uint256 i = 0; i < proposal.signatures.length; ++i) {
-            uint256 quorum = actionsQuorums[proposal.targets[i]][
-                proposal.signatures[i]
-            ] == 0
-                ? DEFAULT_ACTION_THRESHOLD
-                : 0;
+            uint256 quorum = 0;
```

Comment:
question: this syntax  (what it looks like) has change, have the semantics (what it is doing, it's meaning) also changed? 

There seem to be no changes or additional tests. Does this mean the logic of the before & after are the same, or have untested branches been added?

Reply:
It didn't changes logic, just semantics, but only adds an option to compare against protocol.quorum in case if quorum was not set for this action (action.quorum), then we use protocol that has it (contract target address).

Reply:
This chunk is gets tests in tests.